src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,116 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    static final int DEFAULT_INITIAL_CAPACITY = 16;
+    static final float DEFAULT_FACTOR = 0.75f;
+    private int currentCapacity;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int size;
+
+    {
+        this.table = new Node[DEFAULT_INITIAL_CAPACITY];
```

Comment:
Using the diamond operator without type parameters is not recommended. Use 'new Node<>' instead of 'new Node' to avoid unchecked assignment warnings.